FirebaseRemoteConfigSwift/Sources/FirebaseRemoteConfigValueDecoderHelper.swift

```diff
@@ -18,6 +18,11 @@ import Foundation
 import FirebaseRemoteConfig
 import FirebaseSharedSwift
 
+public enum RemoteConfigValueDecoderError: Error {
+  case arrayDecodeFailure
```

Comment:
Just double check for the error should the first letter be capitalized?

Do we need API review for this?

Reply:
Good catch! Removed from the public API surface

Reply:
And now completely removed in favor of using the existing DecodingError